boards/arm/bl5340_dvk/bl5340_dvk_cpuapp_common.dtsi

```diff
@@ -110,7 +110,7 @@
 
 	mipi_dbi {
 		compatible = "zephyr,mipi-dbi-spi";
-		reset-gpios = <&gpio0 6 GPIO_ACTIVE_HIGH>;
+		reset-gpios = <&gpio0 6 GPIO_ACTIVE_LOW>;
 		dc-gpios = <&gpio0 12 GPIO_ACTIVE_HIGH>;
```

Comment:
hey @danieldegrasse, sorry for adding to this PR, I just converted one of my board and noticed that dc also changed polarity from the previous convention -- for this one I think it's fine to use it as active high, but if you want to keep it like that it's a good idea to update the migration guide example (https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/releases/migration-guide-3.6.rst?plain=1#L271-L272) and maybe even mention it separately

Reply:
No worries! I noticed that as well, and I made the change in this PR. I can split it into another PR if you think that would be best though. I'll update the commit in this PR to call out the fact that the DC pin changed polarity

Reply:
nah cool to have it here, you have a duplicate Signed-off-by in the commit message though in case you care about it

Reply:
I do, thanks for noticing- I'll fix it